test/unit-test-coverage/lcov-report/index.html

```diff
@@ -20,31 +20,31 @@ <h1>
     </h1>
     <div class='clearfix'>
       <div class='fl pad1y space-right2'>
-        <span class="strong">86.82% </span>
+        <span class="strong">Unknown% </span>
```

Comment:
What's going on with this coverage report. Also, the coverage report should be ignored and not committed, shouldn't it?